www/service-worker.js

```diff
@@ -0,0 +1,11 @@
+self.addEventListener('activate', function (event) {
```

Comment:
I'm not using it at all, it comes along as a part of starting project through command line (CLI). In this scenario, it would be something like "ionic start upcoming-moviews blank".
Service Workers are like a pocket knife of functionalities that make easier to work Offline through features such as caching, background sync, push notifications and security requirements in the implementation.